src/main/java/de/tum/in/www1/artemis/security/lti/Lti13LaunchFilter.java

```diff
@@ -66,6 +67,10 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
             try {
                 // here we need to check if this is a deep-linking request or a launch request
                 if ("LtiDeepLinkingRequest".equals(ltiIdToken.getClaim(Claims.MESSAGE_TYPE))) {
+                    // Manually setting the deep linking path is required due to Moodle and edX's inconsistent deep linking implementation.
+                    // Unlike standard GET request-based methods, these platforms do not guarantee a uniform approach, necessitating
+                    // manual configuration to ensure reliable navigation and resource access compatibility.
+                    targetLink = CustomLti13Configurer.LTI13_DEEPLINKING_REDIRECT_PATH;
```

Comment:
I think it's good to use a constant here - perhaps the string `"LtiDeepLinkingRequest"` above should be a constant as well (it's used a few times).

Reply:
Thanks for the review @pzdr7, "LtiDeepLinkingRequest" is now a constant